sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/ChangeFeedContextClientImpl.java

```diff
@@ -33,6 +35,8 @@
  * Implementation for ChangeFeedDocumentClient.
  */
 public class ChangeFeedContextClientImpl implements ChangeFeedContextClient {
+    private final Logger logger = LoggerFactory.getLogger(ChangeFeedContextClientImpl.class);
```

Comment:
logger can be static final ?

Reply:
In the other SDK code it looks like we both "private final" and "private static final". Let me know which one we should use one or the other in consistent manner.